client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
```

Comment:
Maybe check in with a designer to see if we're OK using so many custom breakpoints.